{{ cookiecutter.repo_name }}/.secrets

```diff
@@ -1,15 +1,20 @@
 # Secrets and credentials should be stored here as environmental variables. For example:
 #
 #   # Google Cloud authentication credentials
-#   export GOOGLE_APPLICATION_CREDENTIALS="path/to/credentials.json"
+#   GOOGLE_APPLICATION_CREDENTIALS=path/to/credentials.json
```

Comment:
Why has this changed? Is this difference due to moving from .envrc to .env?

Reply:
So I believe this is just because of the sed command which was previously used but as I can't test this it is just an assumption. But the code works and you can import secrets environment variables with the new format given.